test/minlog.test.js

```diff
@@ -0,0 +1,42 @@
+import _ from 'lodash';
+import MinLog from '../src/minlog';
+import defaultLevels from '../src/levels';
+
+describe('MinLog', () => {
+  it('should be defined', () => {
+    expect(MinLog).toBeDefined();
+  });
+
+
+  it('should have default log levels funcs', () => {
+    let instance = new MinLog();
+
+    _.forEach(defaultLevels, (level, levelName) => {
```

Comment:
don't assume that MinLog's default levels are what you import
loop over instance.levels
you can even create a property-testing-wannabe loop (no config, config with custom levels like below, etc): given a series of MinLog configuration, the following test should hold true